templates/html/.tp-config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "parameters": {
+    "sidebarOrganization": {
+      "description": "Defines how the sidebar should be organized. Set its value to 'byTags' to categorize operations by tags.",
+      "required": false
```

Comment:
Yes. I just want people to learn from this file that passing `required` is an option. If you remove `required` from there it will still work because it's checking for `=== true`.